flamingpy/utils/viz.py

```diff
@@ -433,9 +566,9 @@ def _get_node_color(egraph, color_nodes, point):
         color = color_dict.get(node_property)
 
     elif color_nodes == True:
-        color = egraph.nodes[point].get("color") or "k"
+        color = egraph.nodes[point].get("color") or "black"
     else:
-        color = "k"
+        color = "black"
```

Comment:
I would suggest not changing this because it's not relevant to the task at hand and might introduce unwanted changes/bugs

Reply:
But Plotly doesn't recognize 'k' as black.

Reply:
Need to check.

Reply:
The `draw_Egraph_plotly()` method calls the `_get_node_color()` method which returns a list of node colors which is then passed into the plotly's `Scatter3D()` method. So I think changing the color name from `k` to `black` is required. And Matplotlib recognizes both `k` and `black` as black color.

Reply:
Okay I see, great solution in that case! 👍🏼 Sorry for the confusion